examples/helloworld/tests.textpb

```diff
@@ -0,0 +1,35 @@
+env {
+  log_level: TRACE
+}
+test {
+  name: "LoadsConfig"
+  plugin_init {
+    log { regex: ".*Config keys size 3" }
+  }
+}
+test {
+  name: "AllowsGoodToken"
+  request_headers {
+    input { header { key: "User-Token" value: "good-user" } }
+    result { has_header { key: "User-Token" value: "good-user" } }
+  }
+}
+test {
+  name: "DeniesBadToken"
+  request_headers {
+    input { header { key: "User-Token" value: "bad-user" } }
+    result {
+      immediate { http_status: 403 details: "" }
+      body { exact: "Access forbidden.\n" }
+    }
+  }
+}
+test {
+  name: "DeniesMissingToken"
+  request_headers {
+    result {
+      immediate { http_status: 403 details: "" }
+      body { exact: "Access forbidden - token missing.\n" }
+    }
+  }
+}
```

Comment:
Revert this and other tests.textpb files. If we want to use the GCP wasm tester more broadly, we should move it into ProxyWasm first.

Reply:
Aggghhhh unintended changes — accidentally committed these when changing tetratelabs -> proxy-wasm. Done.